rllib/algorithms/impala/impala.py

```diff
@@ -215,6 +217,11 @@ def training(
                 is performing gradient calculations.
             minibatch_buffer_size: How many train batches should be retained for
                 minibatching. This conf only has an effect if `num_sgd_iter > 1`.
+            minibatch_size: The size of minibatches that are trained over during
+                each SGD iteration. Note this only has an effect if
+                `_enable_learner_api` == True.
+                Note: minibatch_size must be a multiple of rollout_fragment_length and
```

Comment:
when computing the vtrace losses, we make the batches time dimension major. In order to do that we need to know ahead of time what the time dimension is of samples in the minibatch. Your minibatch has to contain at least one rollout fragment/sequence and can only be made up of whole sequences. If it isn't, this will break the vtrace computation.

Reply:
got it.